src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -0,0 +1,263 @@
+            <div className={classes.twoColumnGrid}>
+              <FormTextField {...fields.user_name} label="Username" fullWidth />
+              <ApeTextField
+                label="ETH Address"
+                value={myAddress}
+                disabled
+                fullWidth
+              />
+              {myAdminCircles ? (
+                <FormAutocomplete
+                  {...fields.protocol_name}
+                  value={fields.protocol_name.value}
+                  onChange={(v: string) => {
+                    const id = protocols.find((p) => p.name === v)?.id;
+                    fields.protocol_id?.onChange(id);
+                    fields.protocol_name.onChange(v);
+                  }}
+                  options={protocols.map((p) => p.name)}
+                  label="Organization Name"
+                  helperText="Select from list to add to existing org."
+                  fullWidth
+                />
+              ) : (
+                <FormTextField
+                  {...fields.protocol_name}
+                  label="Organization Name"
+                  fullWidth
+                />
+              )}
+              <FormTextField
+                {...fields.circle_name}
+                label="Circle Name"
+                fullWidth
+              />
+            </div>
+            <h3 className={classes.sectionTitle}>Optional Fields</h3>
+            <div className={classes.sectionSubtitle}>
+              Help our product team improve coordinape
+            </div>
+
+            <div className={classes.longform}>
+              <FormTextField
+                label="What compensation needs does your organization have?"
+                {...fields.research_what_needs}
+                multiline
+                fullWidth
+                rows={3}
+              />
+              <FormTextField
+                label="How do you want to use Coordinape?"
+                {...fields.research_how_use}
+                multiline
+                fullWidth
+                rows={3}
+              />
+              <FormTextField
+                label="How is your organization structured and how many people are involved?"
+                {...fields.research_org_structure}
+                multiline
+                fullWidth
+                rows={3}
+              />
+            </div>
+
+            <div className={classes.twoColumnGrid}>
+              <FormTextField
+                {...fields.research_contact}
+                fullWidth
+                label="How we can contact you?"
+              />
+              <FormTextField
+                {...fields.research_org_link}
+                label="Web link"
+                fullWidth
+              />
+              <FormTextField
+                {...fields.research_org_ens}
+                label="Snapshot ENS"
+                fullWidth
+              />
+            </div>
+            <FormHCaptcha {...fields.h_captcha_token} />
```

Comment:
style suggestion: don't put `h` in all the names... unless you're expecting to have multiple captcha implementations in this codebase, it's sufficient to just refer to `captcha` and treat the fact that it's specifically hCaptcha as an implementation detail
(not a merge blocker)

Reply:
Did that